app.py

```diff
@@ -379,6 +379,11 @@ def list_data_bundles(**kwargs):
     return response.marshal()
 
 
+@app.route(base_path + '/service-info', methods=['GET'], cors=True)
+def get_service_info():
+    return {"version": "0.4.0", "name" : "dos-azul-lambda", "description" : "This presents an Amazon Lambda microservice following the Data Object Service. It allows data in the Human Cell Atlas Data Store to be accessed using Data Object Service APIs."}
```

Comment:
This line is too long - PEP8 says that lines should be capped at 80-some characters, but by convention, I think up to 120 is okay. This is still a little long either way - that said, I just shortened the description.

In addition, new requirement - please specify the version as a variable `__version__` near the top of the file, then include it in the response as suggested. (This is also convention that I didn't think to mention when I specified the ticket.)

```suggestion
    return {
        'version': __version__,
        'name': "dos-azul-lambda",
        'description': "Lambda that allows data in the Human Cell Atlas Data Store to be accessed using the Data "
                    "Object Service API. See more at https://github.com/DataBiosphere/dos-azul-lambda"
        }
```